doc/topics/releases/2018.3.2.rst

```diff
@@ -0,0 +1,16 @@
+===========================
+In Progress: Salt 2018.3.2 Release Notes
+===========================
```

Comment:
The overline and underline must match the length of the title line to avoid Sphinx warnings.

Reply:
Sorry, just copied the file from 2018.3.1 and changed the 1 to a 2. Shall I go through and fix all of them?

Reply:
I think it's because the `In Progress:` eventually gets removed when they're ready for release.